bootstrap/terraform/values/prometheus.yaml

```diff
@@ -56,3 +63,11 @@ grafana:
         gnetId: 14584
         revision: 1
         datasource: prometheusdatasource
+      eks:
+        gnetId: 14623
+        revision: 1
+        datasource: prometheusdatasource
+      ekscontrolplane:
+        gnetId: 21192
```

Comment:
nice one! The instances and control plane nodes version can be timeseries count and the version in the legend. With 3 nodes is ok but 300 would be hard to see. Merging this PR is not dependent on that update.